### PR TITLE
added content to the send a message page

### PR DESCRIPTION
**Pull request by stevehenshaw**

### CODE REVIEW

File: cypress/integration/SendAMessagePage.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+    Background:
+        Given I visit the "send message to relatives" page
+
+    Scenario: Has a back link to navigate to the what help are you looking for today page
+        When I click the "back" link
+        Then I am on the "what help are you looking for today" page
+    
+    Scenario: There is a button linking to the Red Cross "Find your local missing family service" page
+        Then I see a "contact your local service for support" button linking to the "redcross find local missing family service" page
```
Comment (WillGibson) [inline cypress/integration/SendAMessagePage.feature:13]: Not a deal breaker, but might be worth setting your IDE to always add an empty line to files when it saves them.

File: src/views/relative-message.html [hunk 2]
```diff
@@ -1 +1,40 @@
-relative-message
\ No newline at end of file
+{% from "components/button.njk" import redcrossButton %}
+{% from "components/progression.njk" import redcrossProgression %}
```
Comment (WillGibson) [inline src/views/relative-message.html]: This line should go.

Comment (WillGibson) [reply src/views/relative-message.html]: I suspect it wants pulling out of the one we did earlier too.

File: cypress/integration/SendAMessagePage.feature (hunk 1 above)
Comment (WillGibson) [reply cypress/integration/SendAMessagePage.feature:13]: I think you might need to tell your IDE to follow our `.editorconfig` file too.

File: src/views/relative-message.html (hunk 2 above)
Comment (stevehenshaw) [reply src/views/relative-message.html]: done

File: cypress/integration/SendAMessagePage.feature (hunk 1 above)
Comment (stevehenshaw) [reply cypress/integration/SendAMessagePage.feature:13]: done (and added the editorconfig extension :) )

**Review by WillGibson — APPROVED** [review]

**Pull request merged**